spi/control-plane/contract-spi/src/main/java/org/eclipse/edc/connector/contract/spi/types/negotiation/ContractRequest.java

```diff
@@ -33,13 +33,11 @@ public class ContractRequest {
     public static final String CONNECTOR_ADDRESS = EDC_NAMESPACE + "connectorAddress";
     public static final String CONTRACT_REQUEST_COUNTER_PARTY_ADDRESS = EDC_NAMESPACE + "counterPartyAddress";
     public static final String PROTOCOL = EDC_NAMESPACE + "protocol";
-    public static final String PROVIDER_ID = EDC_NAMESPACE + "providerId";
```

Comment:
do not delete but mark it as deprecated with the current version ("0.5.1")

Reply:
reverted.